nixos/modules/virtualisation/qemu-vm.nix

```
@@ -769,6 +770,38 @@ in
         '';
       };
 
+    virtualisation.directBoot = {
+      enable =
+        mkOption {
+          type = types.bool;
+          default = !cfg.useBootLoader;
+          defaultText = "!cfg.useBootLoader";
+          description =
+            lib.mdDoc ''
+              If enabled, the virtual machine will boot directly into the kernel instead of through a bootloader. Other relevant parameters such as the initrd are also passed to QEMU.
+
```

```suggestion

This will not boot / reboot correctly into a system that has switched to a different configuration on disk.

```

I suppose this breaks, right?

---

Correct :)